resources/assets/js/src/components/windDirection/WindDirection.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+const WindDirection = (props) => {
+    return (
+        <div className="wind-direction">
+            <img
+                src={`../../../../images/directions/${props.direction}.png`}
```

Comment:
same as previous comment, I have found something like this https://www.npmjs.com/package/app-root-path but you can search for more neat solution. e.g. https://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d